### ISSUE TITLE
RDoc 2.4?

### ISSUE THREAD

**Issue by trans** [mislav/hanna#3]
Any idea when hanna will be able to work with RDoc 2.4+? I would much rather upgrade hanna then downgrade rdoc ;-) Thanks.

**Comment by mislav** [comment]
It already works in the rdoc24 branch, but the template isn't complete. The next release will be mind-blowingly awesome, I'll try to finish it by the end of the month. Until then, you can use last Hanna stable release normally. It doesn't matter if you got RDoc 2.4 installed, Hanna will load an older version of RDoc from RubyGems. (Check its gem dependencies and loading code if you're unsure what I mean.)

Just don't use the "rdoc" command (that loads the latest RDoc)—use "hanna" command or require "hanna/rdoctask" in your Rakefile to create an "rdoc" rake task.

**Comment by gravis** [comment]
not working as expected :)
I had to uninstall rdoc 2.4.3 to have hanna working again

**Comment by mislav** [comment]
It would be helpful if you explained how is it not working and how are you triggering doc generation. Paste any errors, stack traces etc.

**Comment by gravis** [comment]
Yes, really sorry, I didn't want to be impolite (hanna is great, and I'm a huge fan of will_paginate).
I've posted a gist here : http://gist.github.com/188106
then you can see the full "trace" (mostly a "invalid argument: --format=html").

**Comment by mislav** [comment]
Thanks, that helped me realize the problem. I think I nailed it -- try the v0.1.9. (Not sure why GitHub didn't yet rebuild the gem)

**Comment by gravis** [comment]
Yep, Github still haven't buit the gem yet, I've built it from your source.
Anyway, I still have the same error :( 
http://gist.github.com/188398

**Comment by mislav** [comment]
I've messed up the gemspec. Version 0.1.10 is now built.

As for your error; you should have at least Rubygems 1.3.2

**Comment by gravis** [comment]
Thanks, it's working now !

**Comment by rubiii** [comment]
any news on when rdoc 2.4 will be the default? because i heard that version 2.5 is going to be released pretty soon.

**Comment by raggi** [comment]
http://github.com/raggi/hanna my fork supports 2.5, the bulk of the work for that feature was done by erikh
